src/ocean/core/SmartEnum.d

```diff
@@ -111,19 +111,13 @@
 
 module ocean.core.SmartEnum;
 
-
-/*******************************************************************************
```

Comment:
Why all the mixed in formatting changes? :(

Reply:
Because I couldn't decide where to put new import without it :trollface:

Reply:
Hmm... I don't see the relation.